build/gen_stub.php

```diff
@@ -1670,7 +1670,7 @@ public function initializeZval(string $zvalName, iterable $allConstInfos): strin
                 $code .= "\tZVAL_EMPTY_STRING(&$zvalName);\n";
             } else {
                 $constValue = $cConstValue ?: '"' . addslashes($this->value) . '"';
-                $code .= "\tzend_string *{$zvalName}_str = zend_string_init($constValue, sizeof($constValue) - 1, 1);\n";
+                $code .= "\tzend_string *{$zvalName}_str = zend_string_init($constValue, strlen($constValue), 1);\n";
```

Comment:
```suggestion
                $code .= "\tzend_string *{$zvalName}_str = zend_string_init($constValue, strlen($constValue), true);\n";
```

Reply:
@kocsismate why is `1` preferred over `true`

Reply:
@mvorisek I wouldn't say at all that `1` is preferred, but TBH I don't really care which value is used.